core/src/main/java/org/elasticsearch/index/query/HasChildQueryParser.java

```diff
@@ -104,12 +105,20 @@ public static ScoreMode parseScoreMode(String scoreModeString) {
             return ScoreMode.Max;
         } else if ("avg".equals(scoreModeString)) {
             return ScoreMode.Avg;
-        } else if ("total".equals(scoreModeString)) {
+        } else if ("sum".equals(scoreModeString)) {
             return ScoreMode.Total;
         }
         throw new IllegalArgumentException("No score mode for child query [" + scoreModeString + "] found");
     }
 
+    public static String scoreModeAsString(ScoreMode scoreMode) {
+        if (scoreMode == ScoreMode.Total) {
+            return "sum";
```

Comment:
can you leave a comment like "lucene uses 'total' but 'sum' is more consistent with other elasticsearch APIs"?